tests/unit_tests/test_routes.py

```diff
@@ -40,6 +40,20 @@ def test_valid_kml_post(self):
         self.assertEqual(response.content_type, "application/json")  # pylint: disable=no-member
         self.compare_kml_contents(response, self.kml_dict["valid"])
 
+    def test_valid_gzipped_kml_post(self):
+        with open('./tests/samples/valid-kml.xml.gz', 'rb') as kml_file:
+            file = {'kml': (kml_file, 'valid-kml.xml.gz', 'application/vnd.google-earth.kml+xml')}
+            response = self.app.post(
+                url_for('create_kml'),
+                data=file,
+                content_type="multipart/form-data",
+                headers=self.origin_headers["allowed"]
+            )
+        self.assertEqual(response.status_code, 201)
+        self.assertCors(response, ['GET', 'HEAD', 'POST', 'OPTIONS'])
+        self.assertEqual(response.content_type, "application/json")  # pylint: disable=no-member
```

Comment:
pylint no-member could probably be removed ?

Reply:
done